lib/urlwatch/storage.py

```diff
@@ -483,6 +483,20 @@ def load(self, job, guid):
 
         return None, None, 0, None
 
+    def get_history_data(self, guid, count=1):
+        history = {}
+        if count < 1:
+            return history
+        for data, timestamp in CacheEntry.query(self.db, CacheEntry.c.data // CacheEntry.c.timestamp,
+                                                order_by=minidb.columns(CacheEntry.c.timestamp.desc, CacheEntry.c.tries.desc),
+                                                where=(CacheEntry.c.guid == guid)
```

Comment:
Can you instead move the `&` operator to the line starting with `where=`, and remove `W503` from the list of ignored checks?

Reply:
Do you prefer ignoring W504 instead?

W503 and W504 complete opposite of each other, at least one must be ignored. I chose to ignore W503 following [PEP-8's suggestion](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).

Let me know if you do prefer the other way around. Just want to make sure you're aware of this conflict and PEP-8 suggestion.